packages/app/src/client/services/AdminCustomizeContainer.js

```diff
@@ -55,12 +58,18 @@ export default class AdminCustomizeContainer extends Container {
         'tomorrow-night':   { name: '[Dark] Tomorrow Night',  border: false },
         'vs2015':           { name: '[Dark] Vs 2015',         border: false },
       },
+      uploadedLogoSrc: null,
+      defaultLogoSrc: DEFAULT_LOGO,
+      isDefaultLogo: false,
```

Comment:
I guess the initial state value should be true?

Reply:
Set `isDefaultLogo` to true